inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/StreamSinkController.java

```diff
@@ -118,11 +118,14 @@ public Response<Boolean> deleteByKey(@RequestParam(required = false, defaultValu
         return Response.success(sinkService.deleteByKey(groupId, streamId, name, startProcess, username));
     }
 
-    @RequestMapping(value = "/sink/parseFields", method = RequestMethod.POST)
-    @ApiOperation(value = "parse stream sink fields from JSON string")
-    @ApiImplicitParam(name = "fieldsJson", dataTypeClass = String.class, required = true)
-    public Response<List<SinkField>> parseFields(@RequestBody String fieldsJson) {
-        return Response.success(sinkService.parseFields(fieldsJson));
+    @RequestMapping(value = "/sink/parseFields/{statementType}", method = RequestMethod.POST)
+    @ApiOperation(value = "parse stream sink fields from statement")
+    @ApiImplicitParams({
+            @ApiImplicitParam(name = "statement", dataTypeClass = String.class, required = true),
+            @ApiImplicitParam(name = "statementType", dataTypeClass = String.class, required = true)
+    })
+    public Response<List<SinkField>> parseFields(@PathVariable String statementType, @RequestBody String statement) {
```

Comment:
Can we encapsulate all the parameters into an object without displaying the parameters in the path?